config/dev.exs

```diff
@@ -6,8 +6,7 @@ database_url = System.get_env("DATABASE_URL") || "ecto://postgres:postgres@local
 config :chat_api, ChatApi.Repo,
   url: database_url,
   show_sensitive_data_on_connection_error: true,
-  pool_size: 10,
-  socket_options: [:inet6]
+  pool_size: 10
```

Comment:
just curious, why this change? is it necessary for this PR?

Reply:
shoot sorry. I'll revert this. I use it when running the project locally